driver/event_table.c

```diff
@@ -455,8 +455,10 @@ const struct ppm_event_info g_event_info[] = {
 	[PPME_ASYNCEVENT_X] = {"NA", EC_UNKNOWN, EF_UNUSED, 0},
 };
 
+#ifndef __KERNEL__
```

Comment:
It works! My only suggestion is to add a little comment on top of this to explain why this is not needed in kernel mode and the fact that excluding it maximizes compatibility with compilers. Thank you so much!